pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -322,9 +322,17 @@ type BindingSnapshot struct {
 	Clusters []TargetCluster `json:"clusters,omitempty"`
 }
 
-// Suspension defines the policy for suspending of propagation.
+// Suspension defines the policy for suspending of propagation, dispatching and scheduling suspension are independent
+// of each other. The propagation suspension inherited from PropagationPolicy can be transparently transmitted here,
+// and ResourceBinding can also set scheduling suspension separately.
 type Suspension struct {
 	policyv1alpha1.Suspension `json:",inline"`
+
+	// Scheduling controls whether scheduling should be suspended, the scheduler will pause scheduling and not
+	// process resource binding after it's created when value is true and resume scheduling when it's false or nil.
+	// This is usually used for admission checks by third-party such as quota check, multi-tenancy prioritization
+	// and maybe more checks in the future before scheduler really does scheduling.
```

Comment:
```suggestion
	// Scheduling controls whether scheduling should be suspended, the scheduler will pause scheduling and not
	// process resource binding when the value is true and resume scheduling when it's false or nil.
	// This is designed for third-party systems to temporarily pause the scheduling of applications, which enabling
	// manage resource allocation, prioritize critical workloads, etc.
	//
	// It is expected that third-party systems use an admission webhook to suspend scheduling at the time of
	// ResourceBinding creation. Once a ResourceBinding has been scheduled, it cannot be paused afterward, as it may
	// lead to ineffective suspension.
	// +optional
```

Reply:
1. note that scheduling can not be paused after it has been scheduled.
2. +optional